crates/sui-cost/tests/data/dummy_modules_publish/sources/trusted_coin.move

```diff
@@ -16,7 +17,7 @@ module examples::trusted_coin {
     fun init(witness: TRUSTED_COIN, ctx: &mut TxContext) {
         // Get a treasury cap for the coin and give it to the transaction
         // sender
-        let treasury_cap = coin::create_currency<TRUSTED_COIN>(witness, 2, ctx);
+        let treasury_cap = coin::create_currency<TRUSTED_COIN>(witness, 2, b"TRUSTED", b"", b"", option::none(), ctx);
```

Comment:
Am I missing something, where is the coin metadata?

Reply:
This is unit test and it's up to the currency creator to decide whether to leave some fields empty(e.g, some coins might not need a description or logo)